client/app/lib/util/fetchMyRelativeGroups.coffee

```diff
@@ -0,0 +1,27 @@
+_ = require 'lodash'
+kd = require 'kd'
+whoami = require 'app/util/whoami'
+
+###
+  Fetches all groups that are associated with you
+  Removes current group
+  Sort them by group name
+###
+
+module.exports = (callback) ->
+
+  whoami().fetchRelativeGroups (err, groups) ->
+
+    return callback err  if err
+
+    { groupsController } = kd.singletons
+    currentGroup = _.find groups, (group) ->
+      group.slug is groupsController.getCurrentGroup().slug
+
+    rejectedSlugs = [ 'koding', currentGroup.slug ]
```

Comment:
let's handle this case in client version of this call in backend, I mean lets `fetchRelativeGroups$` returns the groups excluded `koding` and current `group` but `fetchRelativeGroups` method will keep return groups as is.